prowler/providers/aws/lib/security_hub/security_hub.py

```diff
@@ -15,34 +15,42 @@
 
 
 def send_to_security_hub(
-    region: str, finding_output: Check_Output_JSON_ASFF, session: session.Session
-):
+    is_quiet: bool,
+    finding_status: str,
+    region: str,
+    finding_output: Check_Output_JSON_ASFF,
+    session: session.Session,
+) -> int:
+    success_count = 0
     try:
-        logger.info("Sending findings to Security Hub.")
-        # Check if security hub is enabled in current region
-        security_hub_client = session.client("securityhub", region_name=region)
-        security_hub_client.describe_hub()
+        # Check if -q option is set
+        if not is_quiet or (is_quiet and finding_status == "FAIL"):
+            logger.info("Sending findings to Security Hub.")
+            # Check if security hub is enabled in current region
+            security_hub_client = session.client("securityhub", region_name=region)
+            security_hub_client.describe_hub()
 
-        # Check if Prowler integration is enabled in Security Hub
-        if "prowler/prowler" not in str(
-            security_hub_client.list_enabled_products_for_import()
-        ):
-            logger.error(
-                f"Security Hub is enabled in {region} but Prowler integration does not accept findings. More info: https://github.com/prowler-cloud/prowler/#security-hub-integration"
-            )
+            # Check if Prowler integration is enabled in Security Hub
+            if "prowler/prowler" not in str(
+                security_hub_client.list_enabled_products_for_import()
+            ):
+                logger.error(
+                    f"Security Hub is enabled in {region} but Prowler integration does not accept findings. More info: https://github.com/prowler-cloud/prowler/#security-hub-integration"
+                )
```

Comment:
Why this if we are always sending findings to Security Hub?

Reply:
We need to check if Prowler Integration is enabled.

Reply:
Yes but if it is not enabled we are sending the findings anyway, is that correct?

Reply:
You are right! I corrected it.